daemon/common.h

```diff
@@ -41,6 +41,9 @@
 // the netdata webserver(s)
 #include "web/server/web_server.h"
 
+// the new h2o based netdata webserver
```

Comment:
We have `#ifdef ENABLE_HTTPD` inside `daemon/main.c` and `daemon/static_threads.c`, should we not add an `ifdef` here to show a clear correlation between this header and necessary `CFLAG`?

Reply:
true